### ISSUE TITLE
Regression on MOD_2.0 on Blackview BL8000

### ISSUE THREAD

**Issue by Evengard** [Displax/safetynet-fix#47]
**Describe the bug**
The 2.4.0_MOD_2.0 doesn't pass CTS and shows HARDWARE attestation level.
The 2.4.0_MOD_1.3 do pass CTS and shows BASIC attestation level.

Looks like a regression. I'm sticking with MOD_1.3 for now.

**To reproduce**
1. Install 2.4.0_MOD_2.0 on a BL8000
2. Run YASNAC, notice HARDWARE attestation and failure to pass CTS
3. Install 2.4.0_MOD_1.3 on the same phone
4. Run YASNAC, notice BASIC attestation and success to pass CTS

**Expected behavior**
The same behaviour on MOD_2.0 is expected as in MOD_1.3

**Device info**
Device model: Blackview BL8000
Android version: 13
ROM name/version: BL8000_S1_20240108V10

**Checklist**
- [X] I confirm that the SELinux status is `Enforcing` on my device.
- [ ] All information is present
- [ ] Screenshots are attached
- [ ] Logs are attached

**Comment by HarriBuh** [comment]
Lol. Looks like you haven't noticed, but this repo is D E A D since last year.
If you are still interested in current development you can head over to forks like [https://github.com/chiteroman/PlayIntegrityFix](url)

**Comment by Evengard** [comment]
Yep, I found out about it after I posted that issue. Old habits die hard...

Migrated to PIF, works fine so far
